moveit_core/robot_state/src/conversions.cpp

```diff
@@ -215,6 +216,15 @@ static void _attachedBodyToMsg(const AttachedBody& attached_body, moveit_msgs::A
       sv.addToObject(sm, p);
     }
   }
+  aco.object.frame_names.clear();
```

Comment:
add comments for what this new code does

Reply:
Does it really belong there? This just implements the message. I think comments explaining it should go in the message definition.